Client/src/Pages/Settings/index.jsx

```diff
@@ -202,17 +208,30 @@ const Settings = ({ isAdmin }) => {
               />
               <Box>
                 <Typography>Clear all stats. This is irreversible.</Typography>
-                <LoadingButton
+                <Button
                   variant="contained"
-                  color="error"
-                  loading={isLoading || authIsLoading || checksIsLoading}
-                  onClick={handleClearStats}
+                  color="error"                  
+                  onClick={()=>setIsOpen({delSts: true, delMntrs: false})}
                   sx={{ mt: theme.spacing(4) }}
                 >
                   Clear all stats
-                </LoadingButton>
+                </Button>
               </Box>
             </Stack>
+            <Dialog
+              arialabelledby="model-clear-stats"
+              ariadescribedby="clear-stats-confirmation"
+              open={isOpen.delSts}
+              onClose={() => setIsOpen({ delSts: false, delMntrs: false })}
+              bdy1="Do you want to clear all stats?"
+              yBtnLbl="Yes, clear all stats"
+              yBtnOnClick={handleClearStats}
+              nBtnLbl="Cancel"
+              nBtnOnClick={() => setIsOpen({ delSts: false, delMntrs: false })}
```

Comment:
setting both to false is repeated on line 275. I suggest extracting that to a function so it can be reused